cmake/pika_setup_vc.cmake

```diff
@@ -87,4 +87,8 @@ if(PIKA_WITH_DATAPAR_VC AND NOT TARGET Vc::vc)
   pika_info("Found Vc (vectorization):" ${Vc_INCLUDE_DIR} "- version:"
             ${Vc_VERSION_STRING}
   )
+
+  if(NOT PIKA_FIND_PACKAGE)
+    target_link_libraries(pika_base_libraries INTERFACE Vc::vc)
```

Comment:
How was it working before? ^^'

Reply:
This was (and still is) linked directly to the module that uses Vc as well. This makes it easier (and possible) to use the same precompiled headers for all modules (Vc sets architecture-specific flags for consumers).